scripts/setup/bootstrap.sh

```diff
@@ -75,10 +75,14 @@ _bootstrap_or_activate() {
         unset PW_CONFIG_FILE
     fi
 
+    declare -a minimum_required_modules=("third_party/nlunit-test/repo" "third_party/nlassert/repo" "third_party/pigweed/repo" "third_party/editline/repo" "third_party/jsoncpp/repo" "third_party/libwebsockets/repo" "third_party/nlio/repo" "third_party/perfetto/repo")
+
     if [ ! -f "$_CHIP_ROOT/third_party/pigweed/repo/pw_env_setup/util.sh" ]; then
```

Comment:
I thought this was the check that was supposed to check for "targeted bootstrap for specific platforms".  That is, because the "pigweed" module is not platform-constrained in `.gitmodules`, running `scripts/checkout_submodules.py` will always check it out.  And then `third_party/pigweed/repo/pw_env_setup/util.sh` will exist.

So it's odd if running bootstrap after running `checkout_submodules.py` ends up doing anything with submodules...

Reply:
Oddly this saves size, but takes longer to cleanup. Going to close this for now and pick it up later.